docs/synapse/httpapi.ipynb

```diff
@@ -448,7 +448,25 @@
     "    *Returns*\n",
     "        On successful upload, or if the file already existed, the API returns information about the file::\n",
     "        \n",
-    "            (size, SHA-256)\n",
+    "            {\n",
+    "              \"md5\": \"<the md5sum value of the uploaded bytes>\",\n",
+    "              \"sha1\": \"<the sha1 value of the uploaded bytes>\",\n",
+    "              \"sha256\": \"<the sha256 value of the uploaded bytes>\",\n",
+    "              \"sha512\": \"<the sha512 value of the uploaded bytes>\",\n",
+    "              \"size\": <the size of the uploaded bytes>\n",
```

Comment:
missing quotes around <the size of the uploaded bytes>

Reply:
it's not a string value returned

Reply:
good point